src/config/variables.go

```diff
@@ -105,17 +108,20 @@ func InjectImportedConstant(importedConstant types.ZarfPackageConstant) {
 	}
 }
 
-func promptVariable(varName string, varDefault string) (string, error) {
-	var value string
+func promptVariable(variable types.ZarfPackageVariable) (value string, err error) {
 
 	pterm.Println()
 
+	if variable.Description != "" {
+		message.Note(variable.Description)
+	}
+
 	prompt := &survey.Input{
-		Message: "Please provide a value for '" + varName + "'",
-		Default: varDefault,
+		Message: "Please provide a value for '" + variable.Name + "'",
```

Comment:
I know this is just pretty colors, but something like this is more pleasing/theme fitting than `message.Note`:

```
description := ""
	if variable.Description != "" {
		description = pterm.FgMagenta.Sprintf("\n - %s", variable.Description)
	}

	prompt := &survey.Input{
		Message: fmt.Sprintf("Please provide a value for '%s'%s", variable.Name, description),
		Default: variable.Default,
	}
```
At the very least we should change this to `fmt.Sprintf()` though

Reply:
I'm totally fine changing it,I try to avoid direct color calls vs function wrappers but I'll look at it.